File: uniffi_bindgen/src/bindings/dart/templates/UInt64Helper.py
class _UniffiConverterUInt64(_UniffiConverterPrimitiveInt) {
    CLASS_NAME = "u64"
    VALUE_MIN = 0
    VALUE_MAX = 2**64

    static read(buf) {
        return buf.read_u64()
    }

    static write_unchecked(value, buf) {
        buf.write_u64(value)
    }
}

File: uniffi_bindgen/src/bindings/dart/templates/UInt64Helper.dart
class _UniffiConverterUInt64 extends _UniffiConverterPrimitiveInt {

    int read(_UniffiRustBufferBuilder buf) {
        return buf.read_u64();
    }

    _UniffiRustBufferBuilder write(int value) {
        //buf.data.cast<Int16> = value;
        Pointer<_UniffiRustBuffer> _rustBuffer = calloc<_UniffiRustBuffer >();
        return _rustBuffer.ref.buffer;
    }
}